docs/docs/configuration/env-variables.md

```diff
@@ -3,100 +3,84 @@
+- **Description**: A list of hostnames that Mathesar will be accessible at ([see Django docs](https://docs.djangoproject.com/en/4.2/ref/settings/#allowed-hosts)). 
+    - Hostnames should not contain the protocol (e.g. `http`) or trailing slashes. 
+    - You can use `localhost` in this list.
+- **Format**: Comma separated string of hostnames
 
     !!! success "Valid values"
         - `mathesar.example.com, localhost`
         - `.localhost, mathesar.example.com, 35.188.184.125`
 
     !!! failure "Invalid values"
-        - `http://mathesar.example.com/` # Contains http protocol and a trailing slash
-        - `https://mathesar.example.com` # Contains http protocol
-        - `localhost/, 35.188.184.125` # Contains trailing slashes, so `localhost` will be invalid.
-
-
+        - `http://mathesar.example.com/` - contains HTTP protocol and a trailing slash
+        - `https://mathesar.example.com` - contains HTTPS protocol
+        - `localhost/, 35.188.184.125` - contains trailing slash after `localhost`
 
 ### `DJANGO_DATABASE_URL`
 
-- _**Required**_
-- A Postgres connection string of the database which will be used for storing metadata related to Mathesar. 
-- The database should **already exist** and won't be created automatically.
-- It should be in the following format:
-
-    ```
-    postgres://user:password@hostname:port/database_name
-    ```
+- **Description**: A Postgres connection string of the database used for **Mathesar's internal usage**. 
+- **Format**:`postgres://user:password@hostname:port/database_name`
+    - The connection string above will connect to a database with username `user`, password `password`, hostname `mathesar_db`, port `5432`, and database name `mathesar_django`.
 
-    > **Example:**
-    >
-    > ```
-    > postgres://mathesar:password@mathesar_db:5432/mathesar_django
-    > ```
-    >
-    > The connection string above will connect to a database with the username `mathesar`, the password `password`, the hostname `mathesar_db`, the port `5432`, and the database name `mathesar_django`.
+### `MATHESAR_DATABASES` 
 
-### `MATHESAR_DATABASES`
+- **Description**: Names and connection information for databases managed by Mathesar. These databases will be accessible through the UI.
+- **Format**:`(unique_id|connection_string),(unique_id|connection_string),...` 
+    - e.g. `(db1|postgresql://u:p@example.com:5432/db1),(db2|postgresql://u:p@example.com:5432/db2)`
+    - This would set Mathesar to connect to two databases, `db1` and `db2` which are both accessed via the same user `u`, password `p`, hostname `example.com`, and port `5432`.
 
-<!-- TODO -->
 
-- _**Required**_
-- Specifies the external databases to be managed by Mathesar.
-- Mathesar will attempt to create databases for any that do not already exist. For this creation to succeed, you will need to ensure that the specified database user has [**`SUPERUSER` Privileges**](https://www.postgresql.org/docs/current/role-attributes.html).
```

Comment:
Was this sentence removed intentionally?

Reply:
Yes, it was. This is not true for installations using build from source, and as discussed on Matrix, I don't want this page to contain information that's specific to a subset of installation types.